packages/react-table/src/components/Table/base/types.tsx

```diff
@@ -81,13 +81,17 @@ export interface TdExpandType {
   columnIndex?: number;
   /** On toggling the expansion */
   onToggle?: OnCollapse;
+  /** Id prefix for expandable rows **/
+  expandId?: string;
```

Comment:
Not a blocker: since it looks like this prop is mainly being used to set an ID on toggles, an alternative name could be `toggleId`.

Reply:
I borrowed this name from the `expandId` prop on `Table.tsx` which also sets this value on the legacy tables. There was not a way to set the equivalent value on the ComposableTable.